inc/storefront-template-functions.php

```diff
@@ -159,14 +159,20 @@ function storefront_header_widget_region() {
 	 * @return void
 	 */
 	function storefront_site_branding() {
-		if ( function_exists( 'the_custom_logo' ) && has_custom_logo() ) {
-			the_custom_logo();
+    if ( function_exists( 'the_custom_logo' ) && has_custom_logo() ) {
+      $logo = get_custom_logo();
+
+      echo $logo = is_home() ? '<h1>' . $logo . '</h1>' : $logo;
```

Comment:
It makes perfect sense to wrap the logo in `<h1>` on home page. Especially that there is not category title to wrap multiple posts in.

Carefull the `<h1>` tag on custom logo has to be styled or wrapped in something. It actually takes the full col width.
It might also break something in header layouts of Storefront Designer.